src/pages/GameScreen/GameScreen.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { GameComponent } from '../../logic/Game/index';
+import PlayerCard from '../../components/PlayerCard/PlayerCard';
+
+export default function GameScreen() {
+  enum PlayerType {
+    bot="bot",
+    player="player",
+    name="Jason",
+ }
```

Comment:
❗️
Я выше писал про то, что хорошо бы иметь `enum` с этими типами)
Давай его отсюда и экспортнём, либо из .`/types`, если вы выбрали это дефолтным расположением типов
```suggestion
export enum PlayerType {
    bot = 'bot',
    player = 'player',
    name = 'Jason',
 }
export default function GameScreen() {
```

Reply:
Вынес в types.ts